CMakeLists.txt

```diff
@@ -0,0 +1,336 @@
+# minimum required vertion of cmake
+cmake_minimum_required (VERSION 3.2)
+
+# project name
+project (raqm)
+
+##############################3
+
+#enable compiler warning for GNU C
+if(CMAKE_COMPILER_IS_GNUCC)
+add_definitions (-Wall  -Wno-portability)
+#enable C99 mode
+add_definitions(-std=gnu99)

```

Comment:
With autotools we check if the compiler supports C99 and what switch it needs, if any. This hard-coding is ugly.